### DISCUSSION TITLE
Support for PDF Viewer

### DISCUSSION THREAD

**Discussion by srinhari** [ProjectMirador/mirador#3559] [General]
Is there a way I can my PDF Viewer to be integrated with the Mirador Viewer? I basically have a React component over a plain PDF.js, that I now want to integrate as one of the viewers within Mirador.

**Comment by enriquediaz** [comment]
It seems that this can technically be achievable through rendering functions, but it's not recommended. This would be considered an enhancement to Mirador, but at this point in time that caliber of work has paused while the contributor community focuses on updating Mirador's core dependencies, such as React. Once that work is done, attention can circle back via community-based discussions about Mirador’s roadmap. Please feel free to join us at one of our [upcoming community calls](https://docs.google.com/document/d/1wWozmLxUsN7QRWzekiIMjjuShhDIsJw0P0lIaiARSa8/edit?usp=sharing) when you’re available!
